blocks/gmo-campaign-details/gmo-campaign-details.js

```diff
@@ -781,37 +394,48 @@ function buildHeaderRow(category, headerType, isInactive) {
     return headerRow;
 }
 
-function buildTableRow(campaignJson, createHidden) {
-    //console.log(campaignJson);
+function buildTableRow(deliverableJson, kpi, createHidden) {
     const dataRow = document.createElement('div');
     dataRow.classList.add('row', 'datarow');
     if (createHidden) dataRow.classList.add('inactive');
+    const status = (deliverableJson.deliverableStatusUpdate == null) ? "??" : deliverableJson.deliverableStatusUpdate + "%";
+    let platformString = '';
+    deliverableJson.platforms?.forEach((platform) => {
+        platformString = platformString + platform + ', ';
+    })
+    platformString = platformString.slice(0, -2);
     dataRow.innerHTML = `
-        <div class='property table-column column1'>${campaignJson.name}</div>
-        <div class='property table-column column2'>${campaignJson.type}</div>
-        <div class='property table-column column3'>${campaignJson.channel}</div>
+        <div class='property table-column column1 deliverable-name'>${deliverableJson.deliverableName}</div>
+        <div class='property table-column column2'>${deliverableJson.deliverableType}</div>
+        <div class='property table-column column3 platforms'>${platformString}</div>
         <div class='property table-column column4'>
-            <a href="${campaignJson.reviewLinkHref}" class="campaign-link">Review Link</a>
+            <a href="${deliverableJson.reviewLink}" class="campaign-link" target="_blank">Review Link</a>
         </div>
         <div class='property table-column column5'>
```

Comment:
@mdickson-adbe looks Final Asset anchor tag is removed. we have `linkedFolderLink` in the API response to show the link

Reply:
There's some logic farther down that appends the anchor tag if `linkedFolderLink` isn't null- see line 433